src/middlewares/validator.js

```diff
@@ -0,0 +1,31 @@
+const createError = require('http-errors')
+const Ajv = require('ajv')
+const ajvKeywords = require('ajv-keywords')
+
+const ajv = new Ajv({v5: true, $data: true, allErrors: true})
+ajvKeywords(ajv)
+
+module.exports = (inputSchema, outputSchema, options) => ({
```

Comment:
I am currently thinking that, in order to easily allow both `inputSchema` and `outputSchema` to be optional and to be more consistent with other middlewares, we should probably accept an option object that contains `inputSchema` and `outputSchema` as keys.

With this approach we can also expose some other configuration layers for `ajv` (e.g. by having a `ajvOptions` key). In such case we should initialize `ajv` inside the middleware of course.